packages/api/package.json

```diff
@@ -34,7 +34,9 @@
     "@polkadot/keyring": "^1.6.1"
   },
   "devDependencies": {
+    "@cennznet/cennznut-wasm": "^0.0.1-dev.0",
     "@cennznet/wallet": "^0.20.7",
+    "@plugnet/doughnut-wasm": "0.2.0",
```

Comment:
Why not "^0.2.0"?

Reply:
I purposely made it 0.2.0, cause if we change the doughnut-wasm to release a new patch version say 0.2.1 and it has breaking changes..we would need to update API to fix it.. 
we can release new wasm package and have API task to use the new feature of it..

Reply:
0.2.1 shouldn't have any breaking changes. Please see:
Caret allows...
- Backward compatible new functionalities
- Large internal refactor
- Bug fixes
- Deprecation of old functionality (which is still operational)
in https://stackabuse.com/caret-vs-tilde-in-package-json/
This would our own mistake if we introduce a breaking change in doughnut-wasm without proper versioning.